jaxlib/mosaic/gpu/runtime.cc

```diff
@@ -154,6 +156,20 @@ void* mosaic_gpu_module_load(void *data) {
     fprintf(stderr, "cuModuleLoadData failed: %s\n", ptr);
     abort();
   }
+
+  CUdeviceptr ptr = 0;
+  size_t size = 0;
+  // Check if module contains NVSHMEM globals implying NVSHMEM state needs to set
+  if (auto result = cuModuleGetGlobal(&ptr, &size, module,
```

Comment:
nit: no need to bind the result since you don't really use it for anything. Just `cuModuleGetGlobal(...) == CUDA_SUCCESS`